.github/workflows/release.yml

```diff
@@ -60,3 +60,10 @@ jobs:
           package: 'packages/lib/dist/package.json'
           token: ${{ secrets.NPM_TOKEN }}
           access: public
+
+      - name: Deploy Storybook 🚀
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: Storybook
+          token: ${{ secrets.USER_TOKEN }} # GH scret containing a Personal Access Token owned by @axelboc
+
```

Comment:
What about putting the two job steps `Build Storybook` and `Deploy` directly here ?

It would avoid redoing the checkout/install, no ?

Reply:
hmm not a bad idea